build.gradle.kts

```diff
@@ -174,10 +172,3 @@ downloadLicenses {
     includeProjectDependencies = true
     dependencyConfiguration = "compileClasspath"
 }
-
-tasks.named<CreateStartScripts>("startScripts") {
```

Comment:
This was needed on Windows. Is it not needed anymore?

Reply:
Works without, but good point. The problem may reappear again. I've kept the task and updated the comment accordingly.